sources/packages/api/src/database/entities/base-sfas-application.model.ts

```diff
@@ -0,0 +1,24 @@
+import { JoinColumn, ManyToOne, Column } from "typeorm";
```

Comment:
Moved individual related properties to base model to avoid code duplication issue

Reply:
We are doing segregating the entity models at this level so far. I believe that it was done due to SonarCloud, but IMO we should keep in the way that we have been doing and ask @abschwenker to ignore the issue on SonarCloud if it is a false positive.

Reply:
Will get the sonar rule in place and moving forward get rid of such situtation.